db/query.go

```diff
@@ -569,3 +576,14 @@ func (e *EmptyResultIterator) NextBytes() []byte {
 func (e *EmptyResultIterator) Close() error {
 	return nil
 }
+
+// Replace $$activeOnlyFilter placeholder with activeOnlyFilterExpression if activeOnly is true
+// and an empty string otherwise in the channel query statement.
+func replaceActiveOnlyFilter(statement string, activeOnly bool) string {
+	activeOnlyFilterExpression := "AND ($sync.flags IS MISSING OR BITTEST($sync.flags,1) = false)"
```

Comment:
This would be better as a constant, declared with activeOnlyFilter, so that all query strings are defined together above.

Reply:
The activeOnlyFilterExpression is declared as constant along with declared with activeOnlyFilter